apps/prairielearn/package.json

```diff
@@ -154,18 +154,18 @@
     "object-hash": "^3.0.0",
     "on-finished": "^2.4.1",
     "on-headers": "^1.0.2",
-    "openai": "^5.1.0",
+    "openai": "^5.8.2",
     "openid-client": "^5.7.1",
     "p-memoize": "^7.1.1",
     "parse5": "^7.3.0",
     "passport": "^0.7.0",
     "passport-azure-ad": "^4.3.5",
     "pem": "^1.14.8",
-    "pg": "^8.16.0",
+    "pg": "^8.16.3",
     "popper.js": "^1.16.1",
-    "preact": "^10.26.8",
+    "preact": "^10.26.9",
     "preact-render-to-string": "^6.5.13",
-    "prettier": "3.5.3",
+    "prettier": "3.6.2",
```

Comment:
Is there a reason why this is not `^3.6.2`?

Reply:
https://prettier.io/docs/install

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.

This is probably irrelevant with a lockfile. I'm happy to change this in another PR.